generate-commit.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+echo "Pulling meta-balena"
+
+if [ -z $TAG ]; then
+	echo "A \$TAG variable is required with the new supervisor version"
+	exit 1
+fi
+
+if [ ! -d /tmp/meta-balena ]; then
```

Comment:
I would make `/tmp/meta-balena` a varible, and use that through the script. That might not be the place where people want to clone stuff, and would be good to be able to configure things